requirements.txt

```diff
@@ -3,6 +3,5 @@ cmake>=3.16
 py-cpuinfo
 setuptools>=65
 setuptools_scm[toml]>=6.2
---extra-index-url https://download.pytorch.org/whl/cpu
-torch==2.1.0+cpu
-accelerate
\ No newline at end of file
+torch==2.1.0
```

Comment:
why not cpu version

Reply:
cpu version of torch cause the following error
<img width="820" alt="image" src="https://github.com/intel/intel-extension-for-transformers/assets/106130696/1b3c07fb-2e36-42c4-ba91-b70fec0ea69d">

Reply:
please do not change this requirement, you can change intel_extension_for_transformers/neural_chat/requirements.txt

Reply:
changed back,  cpu-version of torch is installed prior to torch, although we install torch in intel_extension_for_transformers/neural_chat/requirements.txt, so I add
pip uninstall torch -y
pip install torch